packages/docs/pages/flex.tsx

```diff
@@ -24,153 +24,225 @@ const ExampleBox: React.FC<{ vertical?: boolean }> = ({ children, vertical }) =>
+                    {/* jsx-to-string:start */}
+                    <Flex>
+                      <FlexItem flexGrow={2} flexOrder={2}>
+                        <ExampleBox>Item 1</ExampleBox>
+                      </FlexItem>
+                      <FlexItem flexOrder={4}>
+                        <ExampleBox>Item 2</ExampleBox>
+                      </FlexItem>
+                      <FlexItem flexGrow={4} flexOrder={1}>
+                        <ExampleBox>Item 3</ExampleBox>
+                      </FlexItem>
+                      <FlexItem flexGrow={1} flexOrder={3}>
+                        <ExampleBox>Item 4</ExampleBox>
+                      </FlexItem>
+                    </Flex>
+                    {/* jsx-to-string:end */}
+                  </CodePreview>
+                </>
+              ),
+            },
+          ]}
+        />
+      </Panel>
+
+      <Panel header="Props" headerId="props">
+        <ContentRoutingTabs
+          id="props"
+          routes={[
+            {
+              id: 'flex',
+              title: 'Flex',
+              render: () => (
+                <FlexPropTable
+                  renderPanel={false}
+                  inheritedProps={
+                    <>
+                      <BoxPropTable collapsible />
+                      <DisplayPropTable collapsible />
+                      <MarginPropTable collapsible />
+                      <PaddingPropTable collapsible />
+                    </>
+                  }
+                />
+              ),
+            },
+            {
+              id: 'flex-item',
+              title: 'FlexItem',
+              render: () => (
+                <FlexItemPropTable
+                  renderPanel={false}
+                  inheritedProps={
+                    <>
+                      <BoxPropTable collapsible />
+                      <DisplayPropTable collapsible />
+                      <MarginPropTable collapsible />
+                      <PaddingPropTable collapsible />
+                    </>
+                  }
+                />
+              ),
+            },
+          ]}
+        />
+      </Panel>
 
-      <PageNavigation items={items} />
+      <Panel header="Do's and Don'ts" headerId="guidelines">
+        <GuidelinesTable
+          recommended={[
+            <>
+              Use <Code primary>Flex</Code> when focusing on content flow.
+            </>,
+            <>
+              Use <Code primary>FlexItem</Code> when children need specific flex properties.
+            </>,
+            <>
+              Use <Link href="https://css-tricks.com/snippets/css/a-guide-to-flexbox/">CSS-Tricks complete guide</Link>{' '}
+              to <Code primary>Flexbox</Code>.
+            </>,
+          ]}
+          discouraged={[
+            <>
+              Don’t use <Code primary>Flex</Code> when focusing on content placement. Use{' '}
+              <NextLink href="grid">Grid</NextLink> instead.
```

Comment:
🍹 I was reading up on the MDN and CSS Tricks documentation of flexbox. They mention some things that might be good to include in this description:
- CSS Tricks pointed out that flexbox should be used for simple layouts while grid should be used for more complex layout.
- MDN made a point to think of flexbox as 1-dimensional, while grid is 2-dimension. I think that's a great way of looking at the two.

Reply:
I was reading up about this as there is disagreement between the dimensional distinction, since both can be used for 1 or 2 dimensions, and a more agreeable distinction is to use grid for content placement (to place things exactly were you want to) while flex is more "flow-ish" in nature.